build.rs

```diff
@@ -21,8 +21,18 @@ fn main() {
     #[cfg(feature = "overwrite")]
     let outdir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap()).join("src/sys");
 
-    bindgen::Builder::default()
-        .header_contents("include-file.h", INCLUDE)
+    let mut builder = bindgen::Builder::default();
+
+    if let Some(path) = env::var("BUILD_IO_URING_INCLUDE_FILE")
+        .ok()
+        .filter(|path| !path.is_empty())
+    {
+        builder = builder.header(path);
+    } else {
+        builder = builder.header_contents("include-file.h", INCLUDE);
```

Comment:
What is the benefit to replacing all of INCLUDE? Isn't it just the path to io_uring.h that we want to give fine grained control to?